packages/ra-ui-materialui/src/field/ReferenceField.js

```diff
@@ -149,14 +165,14 @@ ReferenceField.propTypes = {
     sortBy: PropTypes.string,
     source: PropTypes.string.isRequired,
     translateChoice: PropTypes.func,
-    linkType: PropTypes.oneOfType([PropTypes.string, PropTypes.bool])
-        .isRequired,
+    linkType: PropTypes.oneOfType([PropTypes.string, PropTypes.bool, PropTypes.func]),
+    linkTo: PropTypes.oneOfType([PropTypes.string, PropTypes.bool, PropTypes.func]).isRequired,
```

Comment:
not required, because the old code must still work

Reply:
Well it was the same in the old code. It simply is `edit` by default but the prop must be there right ?

Reply:
you're right, as there is a default value it's OK.